cli/tools/upgrade.rs

```diff
@@ -348,12 +349,26 @@ pub async fn upgrade(upgrade_flags: UpgradeFlags) -> Result<(), AnyError> {
   check_exe(&new_exe_path)?;
 
   if !upgrade_flags.dry_run {
-    match upgrade_flags.output {
-      Some(path) => {
-        fs::rename(&new_exe_path, &path)
-          .or_else(|_| fs::copy(&new_exe_path, &path).map(|_| ()))?;
```

Comment:
This previously didn't do windows specific behaviour necessary that's handled in replace_exe.

Reply:
Oh wait. That doesn't matter because it's not the current running executable... I'm going to revert this.